tests/testlib/s2n_pq_hybrid_test_utils.c

```diff
@@ -47,7 +47,7 @@ int s2n_hybrid_pq_rand_cleanup(void) {
 int s2n_hybrid_pq_entropy(void *ptr, uint32_t size) {
     ENSURE_POSIX(s2n_in_unit_test(), S2N_ERR_NOT_IN_UNIT_TEST);
     notnull_check(ptr);
-    eq_check(size, hybrid_kat_entropy_blob.size);
+    lte_check(size, hybrid_kat_entropy_blob.size);
```

Comment:
It's because we're using custom DRBG callbacks when testing the sikep434r2 C code and assembly code consecutively. In between the test for C and assembly, there are requests for entropy of sizes less than `hybrid_kat_entropy_blob.size`. But as long as that callback is set, they obviously fail the `eq_check`. 

The somewhat obvious fix would be to reset the entropy callbacks back to their defaults, but there isn't currently a way to do that. I wasn't really interested in any more DRBG changes (at least not as part of this PR), so this was an easy solution.